pkgs/development/python-modules/trectools/default.nix

```diff
@@ -0,0 +1,48 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, beautifulsoup4
+, pythonOlder
+, pandas
+, numpy
+, scikit-learn
+, scipy
+, lxml
+, matplotlib
+, sarge
+}:
+
+let
+  bs4 = buildPythonPackage rec {
```

Comment:
I had to add the bs4 package instead of beautifulsoup4 because of the requirements of trectools. If anyone has a better idea of how to solve this issue, please feel free to reach out and correct me.